doc/source/userguide.rst

```diff
@@ -114,15 +114,34 @@ size:
 
 ...
 
-Switching from CPU to GPU
-=========================
+Frontends
+=========
+
+You can choose a frontend via the `frontend=` arguments available in all the Scattering constructors. By default, `pytorch`
```

Comment:
Backticks around Scattering. Also, I think it's just `torch` for the name of the frontend.